ibc-testkit/src/testapp/ibc/applications/transfer/module.rs

```diff
@@ -64,10 +64,10 @@ impl Module for DummyTransferModule {
         &mut self,
         _packet: &Packet,
         _relayer: &Signer,
-    ) -> (ModuleExtras, Acknowledgement) {
+    ) -> (ModuleExtras, Option<Acknowledgement>) {
```

Comment:
Would be great to add a docstring for this method explaining when to return ack vs None for other developers' reference.